scripts/linuxaarch64/CROSS_BUILD_TOOLS/build_cross_gcc.sh

```diff
@@ -0,0 +1,172 @@
+#! /bin/bash
+set -e
+trap 'previous_command=$this_command; this_command=$BASH_COMMAND' DEBUG
+trap 'echo FAILED COMMAND: $previous_command' EXIT
+
+#-------------------------------------------------------------------------------------------
+# This script will download packages for, configure, build and install a GCC cross-compiler.
+# Customize the variables (INSTALL_PATH, TARGET, etc.) to your liking before running.
+# If you get an error and need to resume the script from some point in the middle,
+# just delete/comment the preceding lines before running it again.
+#
+# See: http://preshing.com/20141119/how-to-build-a-gcc-cross-compiler
+#-------------------------------------------------------------------------------------------
+
+export BUILDROOT=$(cd $(dirname $0); pwd -P)
+export SYSROOT=$(dirname $0)/../archlinux
+
+if [ ! -d $SYSROOT ]; then
+	wget http://os.archlinuxarm.org/os/ArchLinuxARM-rpi-2-latest.tar.gz
```

Comment:
This is currently 404 (not found) but I think it's the wrong toolchain for aarch64.